src/html/phetpreview_en-US.html

```diff
@@ -0,0 +1,502 @@
+<html>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/css/foundation.css">
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js"></script>
+    <script src="http://cdnjs.cloudflare.com/ajax/libs/foundation/5.5.2/js/foundation.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.js"></script>
+    
+    <link rel="stylesheet" href="../../src/lib/foundation/css/foundation.min.css" />
+    <link rel="stylesheet" href="../../src/lib/foundation/css/foundation.css" />
+    <link rel="stylesheet" href="../../src/lib/foundation/css/foundation-icons/foundation-icons.css" />
+    <!-- CSS required for the Prefs Editor -->
+    <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/preferences/css/Enactors.css" />
+    <link rel="stylesheet" type="text/css" href="../../src/lib/infusion/src/framework/preferences/css/PrefsEditor.css" />
+    <link rel="stylesheet" type="text/css" href="../../src/css/style.css" />
+    
+	<script>
+	    var keys = [];
+	    var messages = [];
+	    var currentMsg;
+	    $(document).ready(function() {
+	        $( "a" ).addClass( "has-tip tip-top" );
+	        $( "a" ).attr( "title", "tooltip");
+	        $( "a" ).attr( "data-tooltip", "" );
+	        $( "a" ).attr( "aria-haspopup", "true" );
+	        $( "a" ).attr( "data-options", "disable_for_touch:true" );
+	        $(document).foundation();
+	        $(".has-tip").focusin(function(){
+	            Foundation.libs.tooltip.showTip( jQuery($(this)) );
+	            console.log("the message is: " + getMessage(getKey(this), keys, messages));
+	            currentMsg = getMessage(getKey(this), keys, messages);
+	        });
+	        $(".has-tip").blur(function(){
+	            Foundation.libs.tooltip.hide( jQuery( $(this) ) );
+	        });
+	        $.getJSON( "../messages/phetMessages_en-US.json", function( data ) {
+	            $.each( data, function( key, val ) {
+	                keys.push(key);
+	                messages.push(val);
+	            });
+	        });
+	
+	    });
+	    getKey = function (that) {
+	        var key = $(that).attr("id");
+	        return key;
+	    };
+	    getMessage = function (key, keysArray, messagesArray) {
+	        var index = $.inArray(key, keysArray);
+	        return messagesArray[index];
+	    };
+	</script>
+
+  
+    <head>
+        <style>
+        		
+            body {

```

Comment:
The same recommendations go for inline styling material - this needs to go in .css files